### PR TITLE
[RFC] Created PhpStorm inspections settings profile

### PR DESCRIPTION
**Pull request by alongosz**
I've added PhpStorm inspections settings profile, intended for importing by every Contributor of eZ Platform-related packages.

In this PR we can discuss enabling/disabling different settings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ This package contains eZ Platform Coding Standards which are aimed to include pl
 for code style tools, plugins for CI, IDE settings, Git hooks, GitHub templates related
 to contributions, etc.
 
+## PhpStorm inspections
+
+See the [IDE/PhpStorm/PhpStormInspections.xml](./IDE/PhpStorm/PhpStormInspections.xml) for
```
Comment (dabrt) [inline README.md]:
```suggestion
See the [PhpStormInspections.xml](./IDE/PhpStorm/PhpStormInspections.xml) file for
```

File: README.md [hunk 2]
```diff
@@ -4,6 +4,15 @@ This package contains eZ Platform Coding Standards which are aimed to include pl
...
+the PhpStorm code inspections settings profile. The profile can be imported by choosing:
```
Comment (dabrt) [inline README.md]:
```suggestion
the PhpStorm code inspections settings profile. 
To import the profile, click:
```

File: README.md [hunk 3]
```diff
@@ -4,6 +4,15 @@ This package contains eZ Platform Coding Standards which are aimed to include pl
...
+
+```
+File -> Settings -> Editor -> Inspections > Import profile
```
Comment (dabrt) [inline README.md]:
```suggestion
File -> Settings -> Editor -> Inspections -> Import profile
```

File: README.md [hunk 4]
```diff
@@ -4,6 +4,15 @@ This package contains eZ Platform Coding Standards which are aimed to include pl
...
+```
+
```
Comment (dabrt) [inline README.md:16]:
```suggestion


For more information, see the [PhpStorm documentation](https://www.jetbrains.com/help/phpstorm/customizing-profiles.html#47e75).


```

**Review by dabrt — CHANGES_REQUESTED** [review]

File: IDE/PhpStorm/PhpStormInspections.xml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<profile version="1.0">
+  <option name="myName" value="eZ" />
```
Comment (alongosz) [inline IDE/PhpStorm/PhpStormInspections.xml]: Well, that's soo outdated

```suggestion
  <option name="myName" value="Ibexa" />
```

**Review by dabrt — APPROVED** [review]

**Comment by alongosz** [comment]
This is quite outdated and based on internal discussions quite a while ago, it's not the best place to keep IDE-specific config.
Right now the go-to tool to help working with Ibexa DXP is [Ibexa DXP Plugin for PHPStorm](https://plugins.jetbrains.com/plugin/17239-ibexa-dxp)

**Pull request closed without merging**